src/datasets/packaged_modules/fasta/fasta.py

```diff
@@ -0,0 +1,137 @@
+import itertools
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Dict, Iterable, Optional
+from typing import List as ListT
+
+import pyarrow as pa
+from Bio import SeqIO
+
+import datasets
+from datasets.features import Value
+from datasets.table import table_cast
+from datasets.utils.file_utils import xopen
```

Comment:
No need for xopen, you can use regular open (it's replaced by xopen anyway when needed due to an old logic)